### PR TITLE
Switch to xo for linting.

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,10 +13,10 @@
...
   },
   "scripts": {
-    "eslint": "eslint .",
-    "lint": "npm run eslint",
-    "test": "npm run eslint && npm run test:integration",
-    "test:integration": "node tests/integration.js"
+    "lint": "npm run xo",
+    "test": "npm run xo && npm run test:integration",
+    "test:integration": "node tests/integration.js",
+    "xo": "xo"
```
Comment (Johann-S) [inline package.json]: What's the use to add a `xo` script? When you can simply call: `npx xo` and it'll use your local `xo` dev deps?

Comment (XhmikosR) [reply package.json]: Same reason we don't do that for the other scripts.

**Comment by XhmikosR** [comment]
I'm not sure if I'll go with this that's why I have it as draft.

File: package.json (hunk 1 above)
Comment (Johann-S) [reply package.json]: Yep but other scripts are pretty long to write, here your `lint` script is totally equal to your `xo` script which adds no values

**Pull request closed without merging**